docker-compose.yml

```diff
@@ -14,6 +14,18 @@ services:
     env_file:
       - .env
 
+  postgres:
+    image: postgres:12
```

Comment:
Given that people need to initialize their database anyway it is probably also a good moment to bump this to postgres 15 (and do that in other places too).

Reply:
What was the reason for being on PostgreSQL 12.8 in the first place? My guess was that it's related to what version stable distributions use, but Debian Stable (Bullseye) has PostgreSQL 13. 

I'm fine with upgrading to PostgreSQL 15 though!